linux/deploy_vm/rebuild_ubuntu_unattend_install_iso.yml

```diff
@@ -89,13 +89,19 @@
       ansible.builtin.replace:
         path: "{{ unattend_iso_cache }}/grub.cfg"
         regexp: '(.*vmlinuz )(.*)'
-        replace: "\\1autoinstall \\2 console=ttyS0,115200n8 "
+        replace: |-
+          {%- if ubuntu_arch is defined and ubuntu_arch == 'aarch64' -%}\1autoinstall \2
+          {%- else -%}\1autoinstall \2 console=ttyS0,115200n8
```

Comment:
Did you try whether it works without `console=ttyS0,115200n8` on x86 server? If it also works, we can remove it for Ubuntu auto install

Reply:
@keirazhang
I have tried it.
For x86 Server, without "console=ttyS0,115200n8", it will lost the necessary log message in serial-xxx-output.log file.

Thanks